app/requests/gotenberg.request.js

```diff
@@ -33,6 +34,11 @@ async function get(path) {
 async function post(path, formData) {
   const result = await _sendRequest(path, BaseRequest.post, formData)
 
+  // Requests for PDFs are sporadic at best. We are talking weeks between users creating them. So, we haven't wasted
+  // money or energy keeping Gotenberg running in an ECS instance with lots of resource.
+  // Because of this we have found it benefits from a brief pause between requests, which we default to 2 secs.
+  await pause(gotenbergConfig.delay)
```

Comment:
Can we not change it so the ECS instance is only spun up when we need it? If none of these pdfs are needed in the service straight away and we control when we do the sending of these to notify could we not just have a job that spins up the instance then sends all the PDF related requests and then it shuts down after 15 mins of inactivity?

Reply:
Ooh, and just remembered one of the improvements we delivered with this was the ability to preview it before sending. That would have to be dropped if we went down this route.